README.md

```diff
@@ -11,7 +11,7 @@
 To run the project `yarn run dev`.
 
 # Production
-
+    
```

Comment:
Why?

Reply:
Sorry, I might have accidentally pressed enter or tab while reading the readme.md file.

Reply:
Can you fix it please? And request a re-review?

Reply:
Ok